params/config.go

```diff
@@ -353,6 +353,9 @@ type NodeConfig struct {
 
 	// DebugAPIEnabled enables debug api
 	DebugAPIEnabled bool
+
+	// MailServerRegistryAddress is the MailServerRegistry contract address
+	MailServerRegistryAddress string
```

Comment:
Have you tried `common.Address`? It implements `Unmarshaler` interface so hopefully it will work just fine.

Reply:
Yes that would work but I though having it has a string in the configuration would be easier to understand if it's empty `""` instead of having a `[20]byte` with all zeros. What do you think?

Reply:
Make sense as we don't use field tags so it would be strange to put `omitempty` here.